clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/AbstractVeniceMapper.java

```diff
@@ -43,6 +44,7 @@
   BytesWritable keyBW = new BytesWritable(), valueBW = new BytesWritable();
 
   protected AbstractVeniceRecordReader<INPUT_KEY, INPUT_VALUE> veniceRecordReader;
+  protected Optional<AbstractVeniceFilter<INPUT_VALUE>> veniceFilter;
```

Comment:
Any specific reason to wrap `AbstractVeniceFilter` in `Optional`?

Reply:
Simply because I think this filter is optional and not every mapper is required to have one, unless the requirement changes in the future.

Reply:
Why not use null instead?

Reply:
I want to avoid to use `null` if I can. As it literally means, I want to make sure we are aware of this filter is indeed optional and actually it is not initialized in other mappers. If it's present, we can do filtering and possibly chain it with some lambda/stream expression

Reply:
ok